src/auth/auth.ts

```diff
@@ -400,7 +410,11 @@ export const isAzureUser = (): boolean => {
 
 export const loadOidcUser = (user: OidcUser): void => {
   oidcStore.update((state) => ({ ...state, user }));
-  authStore.update((state) => {
+  const tokenClaims: B2cIdTokenClaims = user.profile;
```

Comment:
it looks like this is already instantiated on line 418?

Reply:
same with userId

Reply:
Yeah, let's get rid of the copy on line 418 to avoid shadowing a variable.